addons/pos_restaurant/static/tests/tours/pos_restaurant_tour.js

```diff
@@ -67,6 +113,37 @@ registry.category("web_tour.tours").add("pos_restaurant_sync", {
                 content:
                     "acknowledge printing error ( because we don't have printer in the test. )",
             },
+            checkOrderChanges([]),
```

Comment:
in this case, it may cause indeterminate tests, as the printer is not sent synchronously.